example-data/freetype-inner-2.9.spdx.json

```diff
@@ -0,0 +1,80 @@
+{
+  "SPDXID": "SPDXRef-DOCUMENT",
+  "spdxVersion": "SPDX-2.2",
+  "creationInfo": {
+    "created": "2021-09-18T15:38:51Z",
+    "creators": [
+      "Person: Henrik Sandklef <hesa@sandklef.com>",
+      "Organization: Sandklef GNU Labs"
+    ]
+  },
+  "name": "freetype-2.9",
+  "dataLicense": "CC0-1.0",
```

Comment:
This contradicts the GPL-3.0 labeling in the dep5 of reuse

Reply:
This is still an issue

Reply:
Fixed incoming push

```
Files: example-data/osadl-matrix-mini.csv
Copyright: 2021 Henrik Sandklef <hesa@sandklef.com>
License: GPL-3.0-or-later

Files: example-data/*.json
Copyright: 2022 Henrik Sandklef <hesa@sandklef.com>
License: CC0-1.0
```